setup.py

```diff
@@ -63,7 +63,7 @@ def load_requirement_pins(requirements_file: str):
 
 def get_version_spec(dep: str):
     if dep in requirement_pins:
-        return f">={requirement_pins[dep]}"
+        return f"=={requirement_pins[dep]}"
```

Comment:
Not this, please. We can't have user pins to a very specific torch version.

Reply:
done